t/node/upstream-node-dns.t

```diff
@@ -67,7 +67,7 @@ passed
 
     local utils = require("apisix.core.utils")
     utils.dns_parse = function (domain)  -- mock: DNS parser
-        if domain == "test.com" then
+        if domain == "test2.com" then
```

Comment:
Look like this test now will resolve with `test.com` and return "127.0.0.2" always.
Maybe we can change `test.com` to `test1.com` so we can bypass the /etc/hosts in the CI env.

Reply:
I mean replacing `test.com` in the whole test file.